### DISCUSSION TITLE
How would you implement a searchable accordion of checkboxes in filament?

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#13667] [Help]
### Package

Form builder

### Package Version

latest v3

### How can we help you?

![image](https://github.com/user-attachments/assets/3650bfb2-6f0c-4e00-89df-8f73ad777b97)

The main things I'm looking for

1. Show an message when certain checkboxes are ticked
2. Message has a button to show which checkbox caused it to show
3. Searchable
4. Expand/collapse all
5. Show that some/all checkboxes are ticked
6. Ticking A causing B to also tick, i.e. Ticking `Request data` will also tick `View`
7. Show additional context beside checkbox, i.e. Sensitive permission
8. Checkboxes can be indented

**Comment by leandrocfe** [comment]
I think this would be a custom livewire component

**Reply by bilogic** [reply]
> I think this would be a custom livewire component

yea, and one thing that got to me recently is that: everything we see in filament are actually *blade*, not livewire components... still trying to wrap my head around it.
